client/container.go

```diff
@@ -509,8 +509,7 @@ func (c *Client) ContainerSetEACL(ctx context.Context, table eacl.Table, signer
 		return ErrMissingSigner
 	}
 
-	_, isCIDSet := table.CID()
-	if !isCIDSet {
+	if table.LimitedContainer().IsZero() {
```

Comment:
`table.CID().IsZero()` is more clear, in my opinion

Reply:
this would require interface breakage. I'll leave it `GetCID` then

Reply:
I like `GetCID`